### DISCUSSION TITLE
SSL primer?

### DISCUSSION THREAD

**Discussion by fosteraa** [Lissy93/dashy#318] [Q&A] [answered]
First off - love this app, was just what I was looking for.

Long time Windows admin, comfortable with web serving and certificates. Absolute n00b with Docker. I'll have the signed cert ready to go - is there a quick tutorial on how to config your container to serve up over SSL? I'll probably start on the Windows version of Docker to begin if that matters.

**Comment by UrekD** [comment, marked as answer]
The easiest way would probably be to serve it through a reverse proxy, maybe IIS? I use [NPM container](https://github.com/jc21/nginx-proxy-manager?utm_source=nginx-proxy-manager#quick-setup) (Nginx with a web interface) and DNS server pointing to it.

**Reply by fosteraa** [reply]
Thanks!!

**Comment by lissy93** [comment]
+1 to @UrekD's answer - [NGINX Proxy Manager](https://nginxproxymanager.com/) is the best option. Especially since you can setup SSL for all your running apps from one place.

But I've also updated the [SSL Docs](https://github.com/Lissy93/dashy/blob/master/docs/management.md#ssl-certificates) with instructions on how to pass your own self-signed certificate directly to Dashy, which is another option.
Hope that helps :)

**Reply by fosteraa** [reply]
Thanks!!
